### PR TITLE
Edit mode

### PR DESCRIPTION
**Pull request by konradscheng**
https://app.asana.com/0/1199117989666292/1199132519920619/f

### CODE REVIEW

File: controller.js [hunk 1]
```diff
@@ -254,6 +254,67 @@ module.exports.createDish = (req, res) => {
...
+                message: err.message || "Dish could not be created",
+              });
+            });
+        });
+      });
+
+      return new Promise((res, rej) => {res({})});
+    });
+
+    Promise.all(promises).then((results) => {
```
Comment (cchyung) [inline controller.js:311]: Nice this looks good

File: controller.js [hunk 2]
```diff
@@ -254,6 +254,67 @@ module.exports.createDish = (req, res) => {
...
+      console.log(promises);
+      res.send(menu);
+    });
+  });
+};
+
```
Comment (cchyung) [inline controller.js:317]: We just need to make sure to delete the menu if an error occurs during creation

**Review by cchyung — CHANGES_REQUESTED** [review]
Just add functionality to delete the menu if an error occurs during creation

File: controller.js (hunk 2 above)
Comment (konradscheng) [reply controller.js:317]: fixed

Comment (Tony1023) [reply controller.js:317]: Shall we use a Sequelize transaction for bulk stuff like this?

Comment (cchyung) [reply controller.js:320]: That's true - might make more sense to do that.  I can look into this and update accordingly.

**Pull request merged**